install.sh

```diff
@@ -0,0 +1,2 @@
+apt-get install -y sqlite3 libsqlite3-dev libfmt-dev
```

Comment:
```suggestion
#!/bin/bash
apt-get install -y sqlite3 libsqlite3-dev libfmt-dev
```

Reply:
You may need to check the permissions on the script.  It should be 755 when you pull.
Thanks.

Reply:
Done